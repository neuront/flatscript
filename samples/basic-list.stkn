func write_0_2(x, _, z)
    write(x)
    write(z)

write_0_2([false, true], [0, 1], 3.1415926)

y: 3.14159
x: [y, 2.71828]
write(x)

write([] ++ [])
write([] ++ [0])
write([0] ++ [])
write([0] ++ [0])
z: [1.19]
write(z++x)
