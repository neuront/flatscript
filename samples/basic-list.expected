[ 0, 1 ]
3.14159
[ 3.14159, 2.71828 ]
[]
[ 0 ]
[ 0 ]
[ 0, 0 ]
[ 1.19, 3.14159, 2.71828 ]
